docker/run_bot_task.py

```diff
@@ -31,7 +32,27 @@
 )
 
 
+@contextmanager
+def _setenv(name, value):
+    """set an environment variable temporarily"""
+    old = os.environ.get(name, None)
```

Comment:
`None` is already the default return value, so you don't need it there if you don't want.